safe/definitions/fields.py

```diff
@@ -2234,6 +2238,7 @@
     'key': 'affected_productivity_cost_field',
     'name': tr('Affected Productivity Cost'),
     'field_name': 'affected_productivity_cost',
+    'header_name': tr('Production Cost'),
```

Comment:
So why it's written `affected_productivity_cost` ? CF @ismailsunni with the discussion from this morning on the inasafe channel